src/App.js

```diff
@@ -46,7 +46,7 @@ const routes = [
     createRoute("/organization/offers", withUserAuth(OffersView, ["organization"], {getOffers: () => offersAPI.getOffersFromOrganization(), buttons: hrOfferButtons})),
     createRoute("/login", <LoginForm />),
     createRoute("/register", <RegistrationRouting />),
-    createRoute("*", <div>Page</div>)
+    createRoute("*", <Redirect to="/" />)
```

Comment:
Nie koniecznie zawsze będziemy chcieli rzucać do home page'a. Zostawiłbym to tak, jak jest, po prostu trzeba doklepać 404 - user powinien wiedzieć, że coś się sypnęło. Ogólnie wyobraź sobie, że klikasz aplikuj a tu jeb jesteś na stronie głównej - confusing as fuck.

Reply:
Ten route nie będzie obsługiwał takich sytuacji, bo on odpowiada za przekierowanie ścieżek, które się nie maczują do innych. Ale prawda, tu powinno być przekierowanie na jakąś 404, redirect wrzuciłem bo coś testowałem i mnie denerwowało przenoszenie na page'a